anap1003/assignment1/Q1.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package assignment1;
+
+import java.util.*;
+
+/**
+ *
+ * @author Ana
+ */
+public class Q1 {
+
+    // returns true if the inputs are anagrams
+    private static boolean q1(String o, String r) {

```

Comment:
o and r are not very descriptive, neither is q1.
why not
private static boolean areAnagrams(String firstSentence, String secondSentence)